doc/nrf/ug_thread_tools.rst

```diff
@@ -1,21 +1,40 @@
 .. _ug_thread_tools:
 
 Thread tools
-************
+############
 
 When working with Thread in |NCS|, you can use the following tools during Thread application development:
 
-* `nRF Thread Topology Monitor`_ - This desktop application helps to visualize the current network topology.
 * `nRF Sniffer for 802.15.4 based on nRF52840 with Wireshark`_ - Tool for analyzing network traffic during development.
-* :ref:`ug_thread_tools_wpantund`
+* `nRF Thread Topology Monitor`_ - This desktop application helps to visualize the current network topology.
 * `PySpinel`_ - Tool for controlling OpenThread Network Co-Processor instances through command line interface.
+* :ref:`ug_thread_tools_tbr`
+* :ref:`ug_thread_tools_wpantund`
 
 Using Thread tools is optional.
 
+.. _ug_thread_tools_tbr:
+
+Thread Border Router
+********************
+
+Thread Border Router is a specific type of Border Router device that provides connectivity from the IEEE 802.15.4 network to adjacent networks on other physical layers (such as Wi-Fi or Ethernet).
+Border Routers provide services for devices within the IEEE 802.15.4 network, including routing services for off-network operations.
+
+Typically, a Border Router solution consists of the following parts:
+
+* Application based on the :ref:`thread_architectures_designs_cp_ncp` design or its :ref:`thread_architectures_designs_cp_rcp` variant compatible with the IEEE 802.15.4 standard.
+  This application can be implemented for example on an nRF52 device.
+* Host-side application, usually implemented on a more powerful device with incorporated Linux-based operating system.
+
+|NCS| does not provide the complete Thread Border Router solution.
+For development purposes, you can use `OpenThread Border Router`_ , an open-source Border Router implementation that you can set up either on your PC using Docker or on Raspberry Pi.
+OpenThread Border Router is compatible with Nordic Semiconductor devices.
```

Comment:
I think i would remove this sentence or mention it in other words that Nordic Semiconductor devices can be used as NCP/RCP part (but you have already mentioned that in line 27).